kratos/spatial_containers/point_search.h

```diff
@@ -78,25 +78,41 @@ class PointSearch : public SpatialSearch
       ///@name Operations
       ///@{
         
-      void SearchNodesInRadiusExclusive (
-          ModelPart& rModelPart,
-          NodesContainerType const& InputNodes,
-          const RadiusArrayType & Radius,
-          VectorResultNodesContainerType& rResults,
-          VectorDistanceType& rResultsDistance )
-      {
-          static_cast<TDerived*>(this)->SearchNodesInRadiusExclusiveImplementation(rModelPart,InputNodes,Radius,rResults,rResultsDistance);
-      }
-      
-      void SearchElementsInRadiusInclusive (
-          ModelPart& rModelPart,
-          ElementsContainerType const& InputElements,
-          const RadiusArrayType & Radius,
-          VectorResultNodesContainerType& rResults,
-          VectorDistanceType& rResultsDistance )
-      {     
-          KRATOS_THROW_ERROR(std::runtime_error,"Elemental search is not defined for point based structures","");
-      }
+        /**
+         * Search neighbours for every node in "InputNodes" excluding itself
+         * @param StructureNodes      List of nodes against which the neighbours are searched
+         * @param InputNodes          List of nodes to be searched
+         * @param Radius              List of search radius for every node
+         * @param rResults            Array of results for each node
+         * @param rResultDistance     Array of distances for each result of each node
+         */
+        virtual void SearchNodesInRadiusExclusive (
+            NodesContainerType const& StructureNodes,
```

Comment:
I think this could be a more general name since its in the core

Reply:
Aren't you adding this function in this PR?
At least thats how Git shows it

Just a suggestion though